apps/asap-server/src/entities/team.ts

```diff
@@ -1,5 +1,7 @@
 export interface CMSTeam {
   id: string;
+  created: string;
```

Comment:
Are we using this attribute atm or planning on doing so soon?

Reply:
We aren't using created or updated date currently. The new figma designs have updated date in them. Can't hurt as it is provided by Squidex? Ideally this type would be complete.